CenterForOpenScience/ember-osf-web
# [ENG-4575]  Institution dashboard page re-write

-   Ticket: https://openscience.atlassian.net/browse/ENG-4575
-   Git branch: feature/search-institutions-rewrite

## Purpose

The purpose of these changes is to implement the new search page component for the Institutions Discover page.

## Summary of Changes

-Used the new search-page component to re-implement the Institutions dashboard/discover page

-Added institution header with description and banner for desktop, logo and description for mobile

-Added logic for institution colors and default OSF colors when none provided

-Added an affiliated institution filter applied by default to SHARE queries

## Screenshot(s)

<img width="1628" alt="Pasted Graphic 2" src="https://github.com/CenterForOpenScience/ember-osf-web/assets/82047646/b8c1b024-28fe-4f7b-989b-a7c7d839fc94">

Figure 1: Institution re-write with Mirage default colors on desktop until primary and secondary are provided by the backend API

<img width="408" alt="INSTITUTIONS" src="https://github.com/CenterForOpenScience/ember-osf-web/assets/82047646/7d2a7871-d516-41bb-b763-df5e46468ccd">

Figure 2: Institution re-write mock-up with default colors on desktop

## Side Effects

These changes should not produce any side effects.

## QA Notes

-Do proper colors render for specific institutions? Do default OSF colors render when none are provided?
-Are all components properly displayed? Are any not present?
-Upon search, is only the institution's publications returned?
-Does the page look like the mock-up? Are any elements missing? (search results pending)
-Styling is covered in a separate PR, however, should there be any route-specific CSS to prevent element spillage/overflow?
